apis/binding/v1alpha1/servicebinding_types.go

```diff
@@ -28,73 +28,74 @@ var templates = map[string]string{
 	"lowercase": "{{ .name | lower }}",
 }
 
-// ServiceBindingSpec defines the desired state of ServiceBinding
+// ServiceBindingSpec defines the desired state of ServiceBinding.
 type ServiceBindingSpec struct {
-	// MountPath is the path inside app container where bindings will be mounted
-	// If `SERVICE_BINDING_ROOT` env var is present, mountPath is ignored.
-	// If `SERVICE_BINDING_ROOT` is absent and mountPath is present, set `SERVICE_BINDING_ROOT` as mountPath value
-	// If `SERVICE_BINDING_ROOT` is absent but mounthPath is absent, set   SERVICE_BINDING_ROOT as `/bindings`
-	// When mountPath is used, the file will be mounted directly under that directory
-	// Otherwise it will be under `SERVICE_BINDING_ROOT`/<SERVICE-BINDING-NAME>
+	// Specifies the path inside the app container where bindings will be
+	// mounted.  Setting the environment variable `SERVICE_BINDING_ROOT` will
+	// cause this field to be ignored.  If neither of these are set, this field
+	// defaults to `/bindings`.  This results in the file being mounted under
+	// the directory specified.
```

Comment:
The new description diverted from the implemented behavior

Reply:
I reworded this in both a8100a7 and 71484a9, it should be better now.